docs/nodes/maintain/avalanchego-config-flags.md

```diff
@@ -963,6 +967,10 @@ batch, and will then issue the entire batch at once. The value must be at least
 
 ##### `--snow-avalanche-num-parents` (int)
 
+:::caution
```

Comment:
This isn't deprecated - it was deleted

Reply:
fixed 1fa6ed428ac455d0293f6cd97289fd50fb6764e2